app/models/step_by_step_page.rb

```diff
@@ -63,10 +63,11 @@ def mark_as_published
   end
 
   def mark_as_unpublished
+    now = Time.zone.now
```

Comment:
Since `now` is only being used once, I'd suggest just using `Time.zone.now` directly, e.g.

`draft_updated_at: Time.zone.now,`